sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -101,6 +97,37 @@ internal static class Statsbeat
             "westus3",
         };
 
+        internal Statsbeat(string? connectionString)
+        {
+            var parsedConnectionString = ConnectionStringParser.GetValues(connectionString);
+
+            _statsbeat_ConnectionString = GetStatsbeatConnectionString(parsedConnectionString.IngestionEndpoint);
+
+            // Initialize only if we are able to determine the correct region to send the data to.
+            if (_statsbeat_ConnectionString == null)
+            {
+                throw new InvalidOperationException("Cannot initialize statsbeat");
+            }
+
+            _customer_Ikey = parsedConnectionString.InstrumentationKey;
+
+            s_myMeter.CreateObservableGauge("AttachStatsbeat", () => GetAttachStatsbeat());
+
+            // Configure for attach statsbeat which has collection
+            // schedule of 24 hrs == 86400000 milliseconds.
+            // TODO: Follow up in spec to confirm the behavior
+            // in case if the app exits before 24hrs duration.
+            var exporterOptions = new AzureMonitorExporterOptions();
+            exporterOptions.DisableOfflineStorage = true;
+            exporterOptions.ConnectionString = _statsbeat_ConnectionString;
```

Comment:
Are we going to add an `EnablreStatsBeat` to Options?
If so would want to set that here.

Reply:
This is something I am not sure about. We would have something to disable it. But I don't know if it should be part of public options. Spec suggests using env plus appsettings. (Lets discuss first)